### PR TITLE
Add a way to remove waypoints + fixes

### PR DESCRIPTION
**Pull request by DeDiamondPro**
- added /skytilshollowwaypoint set/remove
- fixed player indicator using color of previous thing
- fixed player indicator drawing a square around the indicator 
- fixed map rendering dark
- added config option for special locations on map

### CODE REVIEW

**Review by Sychic — COMMENTED** [review]
Couldn't you make it one command with arguments?

**Comment by DeDiamondPro** [comment]
> Couldn't you make it one command with arguments?

Since I'm also adding something so you can view them all yeah sure I can do that

File: src/main/kotlin/skytils/skytilsmod/commands/HollowWaypointCommand.kt [hunk 1]
```diff
@@ -21,17 +21,19 @@ package skytils.skytilsmod.commands
 import net.minecraft.client.Minecraft
 import net.minecraft.command.CommandBase
 import net.minecraft.command.ICommandSender
-import net.minecraft.util.BlockPos
-import net.minecraft.util.ChatComponentText
+import net.minecraft.event.ClickEvent
+import net.minecraft.event.HoverEvent
+import net.minecraft.util.*
 import skytils.skytilsmod.features.impl.mining.MiningFeatures
+import tv.twitch.chat.Chat
```
Comment (My-Name-Is-Jeff) [inline src/main/kotlin/skytils/skytilsmod/commands/HollowWaypointCommand.kt]: What's this for?

File: src/main/kotlin/skytils/skytilsmod/commands/HollowWaypointCommand.kt [hunk 2]
```diff
@@ -47,54 +49,155 @@ object HollowWaypointCommand : CommandBase() {
     }
 
     override fun processCommand(sender: ICommandSender?, args: Array<out String>?) {
```
Comment (My-Name-Is-Jeff) [inline src/main/kotlin/skytils/skytilsmod/commands/HollowWaypointCommand.kt]: both shouldn't be null, remove the null safe param type

File: src/main/kotlin/skytils/skytilsmod/features/impl/mining/MiningFeatures.kt [hunk 3]
```diff
@@ -57,6 +58,7 @@ import skytils.skytilsmod.utils.RenderUtil.highlight
 import skytils.skytilsmod.utils.graphics.SmartFontRenderer
 import java.awt.Color
 import java.util.regex.Pattern
+import javax.print.DocFlavor
```
Comment (My-Name-Is-Jeff) [inline src/main/kotlin/skytils/skytilsmod/features/impl/mining/MiningFeatures.kt]: What is this for?

File: src/main/kotlin/skytils/skytilsmod/features/impl/mining/MiningFeatures.kt [hunk 4]
```diff
@@ -18,6 +18,7 @@
 package skytils.skytilsmod.features.impl.mining
 
 import gg.essential.universal.UGraphics
+import kotlinx.coroutines.delay
```
Comment (My-Name-Is-Jeff) [inline src/main/kotlin/skytils/skytilsmod/features/impl/mining/MiningFeatures.kt]: When do you use coroutines?

File: src/main/kotlin/skytils/skytilsmod/features/impl/mining/MiningFeatures.kt [hunk 5]
```diff
@@ -453,13 +458,21 @@ class MiningFeatures {
...
-            cityLoc.drawOnMap(50, Color.WHITE.rgb)
-            templeLoc.drawOnMap(50, Color.GREEN.rgb)
-            denLoc.drawOnMap(50, Color.YELLOW.rgb)
-            minesLoc.drawOnMap(50, Color.BLUE.rgb)
-            balLoc.drawOnMap(50, Color.RED.rgb)
-            fairyLoc.drawOnMap(25, Color.PINK.rgb)
+            //basic render function to render map because the one in RenderUtil renders dark
+            GlStateManager.pushMatrix()
+            GlStateManager.enableRescaleNormal()
+            mc.textureManager.bindTexture(ResourceLocation("skytils", "crystalhollowsmap.png"))
```
Comment (My-Name-Is-Jeff) [inline src/main/kotlin/skytils/skytilsmod/features/impl/mining/MiningFeatures.kt]: maybe make the resource location a constant?

**Review by My-Name-Is-Jeff — CHANGES_REQUESTED** [review]

**Review by My-Name-Is-Jeff — APPROVED** [review]

**Review by Sychic — APPROVED** [review]
lgtm

**Pull request merged**